app/src/main/res/xml/preferences.xml

```diff
@@ -26,6 +26,12 @@
         android:key="Locale.Helper.Selected.Language"
         android:summary="@string/prefs_language_summary"
         android:title="@string/prefs_language" />
+    <Preference
+        android:summary="@string/Donation"
+        android:title="@string/Donate">
+        <intent android:action="android.intent.action.VIEW"
+            android:data="https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en" />
```

Comment:
Please use a `@string` resource for the URL so it can be internationalized

Reply:
Ok @policorn, I will definitely make the changes